react/Noop.tsx

```diff
@@ -0,0 +1,5 @@
+import React, { FunctionComponent } from 'react'
+
+const Noop: FunctionComponent = () => <></>
```

Comment:
Would logging an error to the console be useful? I'm thinking of something similar to what is done in the `flex-layout` app: https://github.com/vtex-apps/flex-layout/blob/master/react/FlexLayoutDummy.js

Reply:
Nope. Check the timeline:

1.  The theme would add those empty blocks
2. When opening the store, no consoles should appear
3. Install the wishlist app
4. Store builder would plug the blocks in the blocks of step 1
5. When opening the store, the blocks of the wishlist are in place

Reply:
Oooh! I see! Awesome then 😄